httpserver/CMakeLists.txt

```diff
@@ -50,6 +50,6 @@ target_link_libraries(httpserver
 	${QT_LIBRARIES}
 )
 
-qt5_use_modules(httpserver Core Network)
+target_link_libraries(httpserver Qt5::Core Qt5::Network)
```

Comment:
It is nicer to add these dependencies in the target_link_libraries block opened just above

Reply:
Good idea. I'll make this modification.